src/wallet/rpcwallet.cpp

```diff
@@ -3929,6 +3930,100 @@ UniValue z_setmigration(const UniValue& params, bool fHelp) {
     return NullUniValue;
 }
 
+UniValue z_getmigrationstatus(const UniValue& params, bool fHelp) {
+    if (!EnsureWalletIsAvailable(fHelp))
+        return NullUniValue;
+    if (fHelp || params.size() != 0)
+        throw runtime_error(
+            "z_getmigrationstatus\n"
+            "Returns information about the status of the Sprout to Sapling migration.\n"
+            "In the result a transactions is defined as finalized iff it has ten confirmations.\n"
```

Comment:
Exactly how many users are likely to realise that `iff` is not a typo? 🙃

Reply:
All of the mathematicians :-p

Reply:
But yes, change to `if and only if`.

Reply:
I will change this to `if and only if`. Also, this should `if and only if it has *at least* ten confirmations`. "at least" is also missing from this definition in the ZIP.